license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/CopyrightAuthorProvider.java

```diff
@@ -40,7 +40,7 @@ public class CopyrightAuthorProvider implements PropertiesProvider {
 
   @Override
   public void init(AbstractLicenseMojo mojo, Map<String, String> currentProperties) {
-    gitLookup = GitLookup.create(mojo.defaultBasedir, currentProperties);
+    gitLookup = GitLookup.create(mojo.legacyDefaultBasedir, currentProperties);
```

Comment:
I would use mojo.workspace.baseDir for all these occurences to avoid using deprecated fields in the code base